custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2064,6 +2064,51 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         },
     };
 
+    vm.mobileChartOptions = {
```

Comment:
can you refactor this so there's a `chartOptionsBase` and then we make a copy of it and override just what is different for web and mobile?